### PR TITLE
fix(release): Fixes on chips and tags

### LINKED ISSUE #1219: [Spark request] Tag surface intent
We received a new Spark request, you can find all details here:

https://jira.ets.mpi-internal.com/browse/DO-1740

### PR DESCRIPTION
**Pull request by soulcramer**
## 📋 Changes

Tags Outlined now have a Surface background to ensure correct legibility
 on difficult surface like photos.
Chip in a horizontal scroll container would have its content size reduced to 0 without the size modifier. As a temp fix I added a max width of 9000 to avoid this issue on screen already using this pattern.

## 🤔 Context

close #1219 

## ✅ Checklist

- [x] I have reviewed the submitted code.
- [x] I have tested on a phone device/emulator.
- [ ] If it includes design changes, please ask for a review `spark-design` GitHub team.

### CODE REVIEW

**Comment by soulcramer** [comment]
@spark-ui-bot spotless

File: gradle.properties [hunk 1]
```diff
@@ -20,7 +20,7 @@
 # SOFTWARE.
 #
 
-version=0.11.0-SNAPSHOT
+version=0.11.1-SNAPSHOT
```
Comment (SimonMarquis) [inline gradle.properties:23]: is this update on purpose? Shouldn't it be splitted into a distinct PR?

File: spark/src/main/kotlin/com/adevinta/spark/components/chips/Chip.kt [hunk 2]
```diff
@@ -235,6 +241,9 @@ public fun Chip(
                         color = contentColor,
                     )
                 }
+                // Fix to prevent chip from having a width of zero when used
+//                // in a scroll container
+                .widthIn(max = 9000.dp)
                 .defaultMinSize(minHeight = ChipDefaults.MinHeight)
```
Comment (SimonMarquis) [inline spark/src/main/kotlin/com/adevinta/spark/components/chips/Chip.kt:247]: Can you cleanup this comment? (remove the duplicated `//`)

File: spark/src/main/kotlin/com/adevinta/spark/components/chips/Chip.kt [hunk 3]
```diff
@@ -326,6 +335,9 @@ public fun ChipSelectable(
                         color = contentColor,
                     )
                 }
+//                // Fix to prevent chip from having a width of zero when used
+//                // in a scroll container
```
Comment (SimonMarquis) [inline spark/src/main/kotlin/com/adevinta/spark/components/chips/Chip.kt:339]: same

File: spark/src/main/kotlin/com/adevinta/spark/components/chips/Chip.kt (hunk 2 above)
Comment (SimonMarquis) [reply spark/src/main/kotlin/com/adevinta/spark/components/chips/Chip.kt:247]: Also, IMO this should probably be extracted as an extension function to be re-used/documented.

Comment (soulcramer) [reply spark/src/main/kotlin/com/adevinta/spark/components/chips/Chip.kt:247]: Will do but this is not something I want to [stay in the codebase](https://github.com/adevinta/spark-android/issues/1238) as this is the quick fix for a hack in Polaris to allow trailing icon in chips.

File: spark/src/main/kotlin/com/adevinta/spark/components/chips/Chip.kt (hunk 3 above)
Comment (soulcramer) [reply spark/src/main/kotlin/com/adevinta/spark/components/chips/Chip.kt]: Done in [fe8c41c](https://github.com/adevinta/spark-android/pull/1236/commits/fe8c41c37836b6173f6030041f5959f7bfb6ab08)

File: spark/src/main/kotlin/com/adevinta/spark/components/chips/Chip.kt (hunk 2 above)
Comment (soulcramer) [reply spark/src/main/kotlin/com/adevinta/spark/components/chips/Chip.kt]: Done in [fe8c41c](https://github.com/adevinta/spark-android/pull/1236/commits/fe8c41c37836b6173f6030041f5959f7bfb6ab08)

File: gradle.properties (hunk 1 above)
Comment (soulcramer) [reply gradle.properties]: Nope it was for a local test 😰 
Fixed in [fe8c41c](https://github.com/adevinta/spark-android/pull/1236/commits/fe8c41c37836b6173f6030041f5959f7bfb6ab08)

**Review by SimonMarquis — APPROVED** [review]

File: spark/src/main/kotlin/com/adevinta/spark/components/chips/Chip.kt (hunk 2 above)
Comment (SimonMarquis) [reply spark/src/main/kotlin/com/adevinta/spark/components/chips/Chip.kt]: @soulcramer I meant to extract it like this

```kotlin
/** Workaround for bug XYZ */
private fun Modifier.fixChipsWhatever() = widthIn(max = 9000.dp)
```

**Pull request merged**